src/styles/blocks/page.scss

```
@@ -0,0 +1,10 @@
+.page {
+
+  &__header {
+    margin-bottom: 50px;
+  }
+
+  &__catalog {
+    margin: auto;
+  }
```

try to add margin-top to catalogue instead of margin-bottom to header for the tests

```suggestion
  &__catalog {
    margin: 50px auto;
  }
```

---

It doesn't work either (